lib/compilers/babel-compiler.js

```diff
@@ -1,4 +1,4 @@
-const babel = require('babel-core')
+const babel = require('@babel/core')
```

Comment:
This makes this PR a breaking change, can you keep the old babel-core in this PR? You can use babel-core bridge (https://github.com/babel/babel-bridge) if you need functionality of babel 7.

Reply:
Sorry, I didn't notice this earlier. Would it be possible for you to review the latest code, as I have changed codebase heavily. I will revert back to babel 6 if you think that's a must